Remove mpTelemetry mp60 test repeats

Yas-Franco opened the pull request:
Replaces the MicroProfile 6.0 and 6.1 mpTelemetry 2.0 test repeats with MicroProfile 7.0 (EE10 and EE11)

Yas-Franco committed:
Remove mpTelemetry mp60 test repeats

yasmin-aumeeruddy commented:
#build
#spawn.fullfat.buckets=io.openliberty.microprofile.telemetry.1.0.internal_fat,io.openliberty.microprofile.telemetry.1.0.internal.container_fat,io.openliberty.microprofile.telemetry.1.0.internal_fat_tck,io.openliberty.microprofile.telemetry.1.1.internal_fat_tck,io.openliberty.microprofile.telemetry.2.0.internal_fat_tck

tevans78 commented on dev/io.openliberty.microprofile.telemetry.1.0.internal.container_fat/fat/src/io/openliberty/microprofile/telemetry/internal/suite/FATSuite.java:
```diff
@@ -80,7 +80,7 @@ public static RepeatTests telemetry10and11Repeats(String serverName) {
 
     public static RepeatTests telemetry20Repeats(String serverName) {
-        return TelemetryActions.repeat(serverName, TelemetryActions.MP14_MPTEL20, TelemetryActions.MP41_MPTEL20, TelemetryActions.MP50_MPTEL20, TelemetryActions.MP60_MPTEL20, TelemetryActions.MP61_MPTEL20);
+        return TelemetryActions.repeat(serverName, TelemetryActions.MP14_MPTEL20, TelemetryActions.MP41_MPTEL20, TelemetryActions.MP50_MPTEL20, MicroProfileActions.MP70_EE10, MicroProfileActions.MP70_EE11);
```
MP Tel 2.0 should run MP70_EE10 in LITE mode ... i.e. that one should be listed first. However, we can fix that in a follow up PR.

Yas-Franco committed:
Update Telemetry 2.0 test repeats

Add concurrent-3.1 to the tested.feeatures in the test bnd for telemetry.1.0.internal_fat

yasmin-aumeeruddy commented:
#libby
